src/v2/view-builder/views/custom-app/ChallengeCustomAppPushView.js

```diff
@@ -0,0 +1,85 @@
+import { _, loc, createButton, View } from 'okta';
```

Comment:
How is this view is different than `ChallengeOktaVerifyPushView` ? This Seems a complete copy.

We should try to generalize instead.. Create a class that can handle both `custom-app` and `okta verify` and all the similar use cases.. Ideally for code there should not be any difference among them right ?

Reply:
The strings are all customizable and different for each view. Also we have a different classname.